docs/content/v0.20/examples.md

```diff
@@ -21,22 +21,22 @@ The following steps establish an initial state used in the examples below.
    ```
    If you are installing KubeStellar on an existing Kubernetes or OpenShift cluster, just use the command `kflex init`.
 
-2. Update the post-create-hooks in KubeFlex to install kubestellar with the v0.20.0 images:
+1. Update the post-create-hooks in KubeFlex to install kubestellar with the v0.20.0 images:
```

Comment:
The numbers of bulletin points 2, 3, 4, 5, 6 seem to all be 1?

Reply:
yes. this is the "github" way of writing README files with numbered items. 
that way when you want to add an item you don't need to modify the all the numbers in the list (e.g., if you add item at the beginning or in the middle).
when presenting the md file in github UI, it will show the numbering for you.
you can see it if you go to "view file" in this PR.